dashboard/app/models/pd/workshop.rb

```diff
@@ -197,15 +197,17 @@ def self.organized_by(organizer)
   end
 
   def self.facilitated_by(facilitator)
-    joins(:facilitators).where(users: {id: facilitator.id}).distinct
+    left_outer_joins(:facilitators).where(users: {id: facilitator.id}).distinct
```

Comment:
Ah right, that. Initially, we had a [test failure](https://circleci.com/gh/code-dot-org/code-dot-org/45107?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) in the case of both `.facilitated_or_organized_by` and `.facilitated_by` in the same query, which can happen now in the [controller](https://github.com/code-dot-org/code-dot-org/blob/75eeeb35f4906dbda9943ebf58352bc2cbc6d4a8/dashboard/app/controllers/api/v1/pd/workshops_controller.rb#L15) for `facilitator_view`.

It turns out Rails doesn't play well with mixing different types of joins on the same association:

Even something as simple as
```ruby
Pd::Workshop.joins(:facilitators).left_outer_joins(:facilitators)
```
raises `ActiveRecord::StatementInvalid: Mysql2::Error: Not unique table/alias`

Whereas
```ruby
Pd::Workshop.left_outer_joins(:facilitators).left_outer_joins(:facilitators)
```
works fine. 

In this case, since we're matching the facilitator id, left outer join is actually more appropriate anyway (though it will have the same results).

Note: another way to get around this would be to alias one of the joins which has to be done in raw SQL, e.g.
```ruby
joins('LEFT OUTER JOIN pd_workshops_facilitators AS scoped_facilitators ON scoped_facilitators.pd_workshop_id = pd_workshops.id').
```

Obviously that is less clean, so I went with consistent joins.